ingenialink/pdo.py

```diff
@@ -310,6 +310,28 @@ def items_mapping(self) -> bytearray:
             map_bytes += pdo_map_item.register_mapping
         return map_bytes
 
+    def set_item_bytes(self, data_bytes: bytes) -> None:
+        """Set the items raw data from a byte array received from the slave.
```

Comment:
Remove `received from the slave`, it is not a must.

Reply:
Done